gulpfile.js

```diff
@@ -56,6 +56,18 @@ function watch() {
   gulp.watch(copyFileSources, copy);
 }
 
+var typedoc = require('gulp-typedoc');
+gulp.task('typedoc', function() {
+  return gulp.src(['src/**/*.ts']).pipe(
```

Comment:
I wasn't able to get this working locally. There are a number of compiler options not being picked up by this plugin (the one causing me the most headache was its ignorance of `allowSyntheticDefaultImports` and `esModuleInterop`). I eventually had to resort to using `execa()` to run the `typedoc` script directly.